kde-frameworks/kpeople/kpeople-5.18.0.ebuild

```diff
@@ -29,5 +29,4 @@ DEPEND="
 "
 RDEPEND="${DEPEND}
 	!net-libs/kpeople:5
-	!net-libs/libkpeople[-minimal(-)]
```

Comment:
Touching stable isn't needed. (-) covers the absence of the use flag. KF 5.21.0 is under stabilization so 5.18.0 will be dropped soon.